esp/src/stub.htm

```diff
@@ -35,26 +35,26 @@
             document.write("<script type=\"text/javascript\" src=\"/esp/files/dist/stub.eclwatch.js\" charset=\"utf-8\"><" + "/script>");
         }
         getAuthType();
-        function getAuthType() {
-            if (document.cookie.indexOf("ESPSessionState=true") > -1) {
+        function getAuthType(){
+            if (document.cookie.indexOf ("ESPSessionState=true") > -1){
                 checkCookie();
-            } else if (document.cookie.indexOf("ESPSessionState=false") > -1) {
-                // window.location();
+            } else if (document.cookie.indexOf ("ESPSessionState=false") > -1) {
+                window.location();
```

Comment:
Chrome will complain this is not a function. Use `window.location.href` instead (previous issue not something you introduced)

Reply:
Can we open a separate Jira to address that please